### PR TITLE
🐛 Source Mongo: Ignore internal collections in discover

### PR DESCRIPTION
**Pull request by jdpgrailsdev**
## What
* Prevent discover failures due to Mongo restriction on access to certain system-level collections.

## How
* Filter out system-level collections when discovering schema

## Recommended reading order
1. `MongoDbSource.java`

### CODE REVIEW

File: airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io.airbyte.integrations.source.mongodb/MongoDbSource.java [hunk 1]
```diff
@@ -109,20 +115,22 @@ protected List<TableInfo<CommonField<BsonType>>> discoverInternal(final MongoDat
...
-          .nameSpace(database.getName())
-          .name(collectionName)
-          .fields(fields)
-          .primaryKeys(List.of(MongoUtils.PRIMARY_KEY))
-          .build();
-
-      tableInfos.add(tableInfo);
-    });
+    authorizedCollections.parallelStream()
+        .filter(this::isSupportedCollection)
```
Comment (colesnodgrass) [inline airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io.airbyte.integrations.source.mongodb/MongoDbSource.java:119]: I wish that java had a [filterNot](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/filter-not.html)

**Review by colesnodgrass — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-mongodb-strict-encrypt test report (commit [`4342d7030c`](https://github.com/airbytehq/airbyte/pull/28799/commits/4342d7030cf3a238d1ee67141972d7d9dd09f3f0)) - ❌

⏲️  Total pipeline duration: 04mn55s 

| Step                                                                                     | Result   |
|:-----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mongodb-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                           | ✅        |
| QA checks                                                                                | ❌        |
| Build connector tar                                                                      | ✅        |
| Build source-mongodb-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mongodb-strict-encrypt:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690478084/4342d7030cf3a238d1ee67141972d7d9dd09f3f0/source-mongodb-strict-encrypt/0.2.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mongodb-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mongodb-v2 test report (commit [`4342d7030c`](https://github.com/airbytehq/airbyte/pull/28799/commits/4342d7030cf3a238d1ee67141972d7d9dd09f3f0)) - ❌

⏲️  Total pipeline duration: 11mn58s 

| Step                                                                         | Result   |
|:-----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mongodb-v2/metadata.yaml     | ✅        |
| Connector version semver check                                               | ✅        |
| QA checks                                                                    | ✅        |
| Build connector tar                                                          | ✅        |
| Build source-mongodb-v2 docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mongodb-v2:integrationTest | ❌        |
| Acceptance tests                                                             | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690478084/4342d7030cf3a238d1ee67141972d7d9dd09f3f0/source-mongodb-v2/0.2.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mongodb-v2 test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mongodb-strict-encrypt test report (commit [`4342d7030c`](https://github.com/airbytehq/airbyte/pull/28799/commits/4342d7030cf3a238d1ee67141972d7d9dd09f3f0)) - ❌

⏲️  Total pipeline duration: 02mn23s 

| Step                                                                                     | Result   |
|:-----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mongodb-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                           | ✅        |
| QA checks                                                                                | ❌        |
| Build connector tar                                                                      | ✅        |
| Build source-mongodb-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mongodb-strict-encrypt:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690479267/4342d7030cf3a238d1ee67141972d7d9dd09f3f0/source-mongodb-strict-encrypt/0.2.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mongodb-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mongodb-v2 test report (commit [`4342d7030c`](https://github.com/airbytehq/airbyte/pull/28799/commits/4342d7030cf3a238d1ee67141972d7d9dd09f3f0)) - ❌

⏲️  Total pipeline duration: 07mn55s 

| Step                                                                         | Result   |
|:-----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mongodb-v2/metadata.yaml     | ✅        |
| Connector version semver check                                               | ✅        |
| QA checks                                                                    | ✅        |
| Build connector tar                                                          | ✅        |
| Build source-mongodb-v2 docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mongodb-v2:integrationTest | ❌        |
| Acceptance tests                                                             | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690479267/4342d7030cf3a238d1ee67141972d7d9dd09f3f0/source-mongodb-v2/0.2.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mongodb-v2 test
```

**Review by colesnodgrass — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-snowflake test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 15mn00s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-snowflake/metadata.yaml     | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build source-snowflake docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-snowflake:integrationTest | ❌        |
| Acceptance tests                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/source-snowflake/0.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-snowflake test
```

**Comment by jdpgrailsdev** [comment]
Closing in favor of https://github.com/airbytehq/airbyte/pull/28815

**Comment by octavia-squidington-iii** [comment]
##  source-cockroachdb test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 305mn27s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-cockroachdb/metadata.yaml     | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Build connector tar                                                           | ✅        |
| Build source-cockroachdb docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-cockroachdb:integrationTest | ❌        |
| Acceptance tests                                                              | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/source-cockroachdb/0.1.22/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-cockroachdb test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-mongodb-strict-encrypt test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ✅

⏲️  Total pipeline duration: 09mn04s 

| Step                                                                                          | Result   |
|:----------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-mongodb-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                                | ✅        |
| QA checks                                                                                     | ✅        |
| Build connector tar                                                                           | ✅        |
| Build destination-mongodb-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-mongodb-strict-encrypt:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-mongodb-strict-encrypt/0.2.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-mongodb-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-alloydb test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 08mn42s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-alloydb/metadata.yaml     | ✅        |
| Connector version semver check                                            | ✅        |
| QA checks                                                                 | ✅        |
| Build connector tar                                                       | ✅        |
| Build source-alloydb docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-alloydb:integrationTest | ✅        |
| Acceptance tests                                                          | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/source-alloydb/3.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-alloydb test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-csv test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ✅

⏲️  Total pipeline duration: 08mn26s 

| Step                                                                       | Result   |
|:---------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-csv/metadata.yaml     | ✅        |
| Connector version semver check                                             | ✅        |
| QA checks                                                                  | ✅        |
| Build connector tar                                                        | ✅        |
| Build destination-csv docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-csv:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-csv/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-csv test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-r2 test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 09mn28s 

| Step                                                                      | Result   |
|:--------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-r2/metadata.yaml     | ✅        |
| Connector version semver check                                            | ✅        |
| QA checks                                                                 | ✅        |
| Build connector tar                                                       | ✅        |
| Build destination-r2 docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-r2:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-r2/0.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-r2 test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-teradata test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ✅

⏲️  Total pipeline duration: 18mn34s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-teradata/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-teradata docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-teradata:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-teradata/0.1.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-teradata test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-e2e-test test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 08mn27s 

| Step                                                                            | Result   |
|:--------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-e2e-test/metadata.yaml     | ✅        |
| Connector version semver check                                                  | ✅        |
| QA checks                                                                       | ✅        |
| Build connector tar                                                             | ✅        |
| Build destination-e2e-test docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-e2e-test:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-e2e-test/0.3.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-e2e-test test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-postgres-strict-encrypt test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 20mn44s 

| Step                                                                                           | Result   |
|:-----------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-postgres-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                                 | ✅        |
| QA checks                                                                                      | ✅        |
| Build connector tar                                                                            | ✅        |
| Build destination-postgres-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization:dev                                                                | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-postgres-strict-encrypt:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-postgres-strict-encrypt/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-postgres-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-clickhouse-strict-encrypt test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 305mn57s 

| Step                                                                                             | Result   |
|:-------------------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-clickhouse-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                                   | ✅        |
| QA checks                                                                                        | ✅        |
| Build connector tar                                                                              | ✅        |
| Build destination-clickhouse-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-clickhouse:dev                                                       | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-clickhouse-strict-encrypt:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-clickhouse-strict-encrypt/0.2.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-clickhouse-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  source-mssql-strict-encrypt test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 307mn22s 

| Step                                                                                   | Result   |
|:---------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-mssql-strict-encrypt/metadata.yaml     | ✅        |
| Connector version semver check                                                         | ✅        |
| QA checks                                                                              | ✅        |
| Build connector tar                                                                    | ✅        |
| Build source-mssql-strict-encrypt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:source-mssql-strict-encrypt:integrationTest | ❌        |
| Acceptance tests                                                                       | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/source-mssql-strict-encrypt/1.1.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-mssql-strict-encrypt test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-kinesis test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ✅

⏲️  Total pipeline duration: 14mn00s 

| Step                                                                           | Result   |
|:-------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-kinesis/metadata.yaml     | ✅        |
| Connector version semver check                                                 | ✅        |
| QA checks                                                                      | ✅        |
| Build connector tar                                                            | ✅        |
| Build destination-kinesis docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-kinesis:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-kinesis/0.1.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-kinesis test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-mqtt test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 24mn09s 

| Step                                                                        | Result   |
|:----------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-mqtt/metadata.yaml     | ✅        |
| Connector version semver check                                              | ✅        |
| QA checks                                                                   | ✅        |
| Build connector tar                                                         | ✅        |
| Build destination-mqtt docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-mqtt:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-mqtt/0.1.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-mqtt test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-pulsar test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ✅

⏲️  Total pipeline duration: 24mn21s 

| Step                                                                          | Result   |
|:------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-pulsar/metadata.yaml     | ✅        |
| Connector version semver check                                                | ✅        |
| QA checks                                                                     | ✅        |
| Build connector tar                                                           | ✅        |
| Build destination-pulsar docker image for platform linux/x86_64               | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-pulsar:integrationTest | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-pulsar/0.1.3/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-pulsar test
```

**Comment by octavia-squidington-iii** [comment]
##  destination-clickhouse test report (commit [`0b7851000f`](https://github.com/airbytehq/airbyte/pull/28799/commits/0b7851000f752fbc7aec7117e5462554d9e8bc6f)) - ❌

⏲️  Total pipeline duration: 306mn06s 

| Step                                                                              | Result   |
|:----------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/destination-clickhouse/metadata.yaml     | ✅        |
| Connector version semver check                                                    | ✅        |
| QA checks                                                                         | ✅        |
| Build connector tar                                                               | ✅        |
| Build destination-clickhouse docker image for platform linux/x86_64               | ✅        |
| Build airbyte/normalization-clickhouse:dev                                        | ✅        |
| ./gradlew :airbyte-integrations:connectors:destination-clickhouse:integrationTest | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/jonathan_mongodb-unsupported-collections/1690484887/0b7851000f752fbc7aec7117e5462554d9e8bc6f/destination-clickhouse/0.2.5/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-clickhouse test
```

**Pull request closed without merging**